packages/generators/init-generator.ts

```diff
@@ -81,15 +83,19 @@ export default class InitGenerator extends Generator {
 			"\n"
 		);
 
+		const initConfig = []
```

Comment:
You already have an object that you have access to already in two ways hint hint

Reply:
So do I pass in the entire object, it contains a lot of other data which is a lot level nested? So I thought to pass in just the options user picked makes it a lot easier to implement the compare function and seems work pretty well(one liner) looks cleaner. WDYT? :) @evenstensberg

Reply:
this.configuration is better than alloc memory for all that

Reply:
Okay then! Changing in a sec ✨ 😄

Reply:
It turns out that not all prompt answers are present in the config and therefore some memory is needed anyways, for now I've shifted the config object inside `this.configuration` and passing the config object from there to check, if we don't want this way we can store the missing things in this.configuration and then check it, but this approach looks more maintainable to me. WDYT about this?

Reply:
My 2 cents, I agree with @anshumanv and I'd put the answers into `this.configuration.answers`.
Iside `this.configurations.config` we have `topScope` and `webpackConfig` that directly influence the generated webpack config file